cmd/jaeger/internal/integration/datareceivers/jaegerstorage.go

```diff
@@ -37,19 +40,41 @@ func NewJaegerStorageDataReceiver(traceStorage string, storageConfig *jaegerstor
 func (dr *jaegerStorageDataReceiver) Start(tc consumer.Traces, _ consumer.Metrics, _ consumer.Logs) error {
 	ctx := context.Background()
 
-	extFactory := jaegerstorage.NewFactory()
-	ext, err := extFactory.CreateExtension(ctx, extension.CreateSettings{
+	zapCfg := &zap.Config{
+		Level:             zap.NewAtomicLevelAt(zapcore.InfoLevel),
+		Development:       false,
+		Encoding:          "console",
+		EncoderConfig:     zap.NewProductionEncoderConfig(),
+		OutputPaths:       []string{"stderr"},
+		ErrorOutputPaths:  []string{"stderr"},
+		DisableCaller:     false,
+		DisableStacktrace: false,
+		InitialFields:     map[string]any(nil),
+	}
+	zapCfg.EncoderConfig.EncodeTime = zapcore.ISO8601TimeEncoder
+	logger, err := zapCfg.Build()
```

Comment:
I think there's a much simpler way, something like `zap.NewDevelopment()`

Reply:
also, it's usually a bad pattern for a component to initialize telemetry, telemetry objects should be injected. The production OTEL components receive TelemetrySettings, which I think you also initialize somewhere at the start of the integration test, so the logger should be set up there.

Reply:
I see, I've refactored it. Actually, the TelemetrySettings is set from the config file at `service.telemetry.logs`, so I've refactored the data receiver to get that setting from the config file and injected the logger.